src/bindings/js/node/tests/infer_request.test.js

```diff
@@ -0,0 +1,112 @@
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+const ov = require('../build/Release/ov_node_addon.node');
+const assert = require('assert');
+const { describe, it } = require('node:test');
+const { getModelPath } = require('./utils.js');
+
+describe('InferRequest', () => {
+
+  let testXml = getModelPath();
```

Comment:
Maybe I forgot why here cannot use const, just ignore if cannot
```suggestion
  const testXml = getModelPath();
```